experimental/waveshare_eink/linux_kernel_modules/mangOH_red_ws213.c

```diff
@@ -27,6 +27,12 @@ static struct waveshare_eink_platform_data ws213_pdata = {
 	.rst_gpio	= CF3_GPIO23,
 	.dc_gpio	= CF3_GPIO22,
 	.busy_gpio	= CF3_GPIO24,
+	.display	=
+	{
+		.width	= 122,
```

Comment:
122 is the visible width, but 128 is the addressable width.  Eventually, both values will need to be part of the settings associated with the device.  I think that this is outside the scope of this change though.  This change should just be about associating the width, height and bpp parameters with the device instance rather than using globals.